src/rgw/rgw_user.h

```diff
@@ -53,6 +53,9 @@ extern int rgw_user_sync_all_stats(RGWRados *store, const rgw_user& user_id);
 /**
  * Get the anonymous (ie, unauthenticated) user info.
  */
+ 
+extern int rgw_user_get_all_buckets_stats(RGWRados *store, const rgw_user& user_id, map<string, cls_user_bucket_entry>&buckets_usage_map);
+
```

Comment:
nit: the block comment above applies to the `rgw_get_anon_user()` declaration below - could you move this declaration?